backend/server/services/bucket_service.ts

```diff
@@ -1,9 +1,10 @@
+import { request } from 'http';
 import logger from '../logger';
 import * as errors from 'feathers-errors';
 import { BucketModel } from '../models/bucket';
 
-class BucketService {
-  find({ userID, bucketID }) {
+export default class BucketService {
+  find({query: {userID, bucketID}}) {
```

Comment:
@dwene here's the destructured version

For some reason find forces a query object and won't let me just pass parameters as I thought I could. I have to pass them through the query object. Not quite sure why.